step.yml

```
@@ -107,7 +107,7 @@ inputs:
   - use_apk_signer: "false"
     opts:
       title: "Enables apksigner"
-      description: "Indicates if the signature should be done using apksigner instead of jarsigner."
+      description: "Indicates if the signature should be done using `apksigner` instead of `jarsigner`."
```

We may standardize the use of ` for copiable text parameters and use italics for tool names.
```suggestion
      description: "Indicates if the signature should be done using *apksigner* instead of *jarsigner*."
```

---

I generally use the `code` syntax for anything coding-related, in this case I changed the formatting because `apksigner` and `jarsigner` are exact CLI commands, not just specific terms.

---

Can live with that, it would be great to have style guide tough for consistency.